timing-entrytypes-registry/registry.any.js

```diff
@@ -0,0 +1,41 @@
+// META: script=resources/utils.js
+
+test(() => {
+  assert_true(!!self.PerformanceObserver, "PerformanceObserver");
+  assert_true(!!self.PerformanceObserver.supportedEntryTypes,
+              "PerformanceObserver.supportedEntryTypes");
+}, "PerformanceObserver.supportedEntryTypes exists");
+
+// UPDATE HERE if new entry
+[
+  [ "mark", "PerformanceMark" ],
+  [ "measure", "PerformanceMeasure" ],
+  [ "resource", "PerformanceResourceTiming" ],
+  [ "longtask", "PerformanceLongTaskTiming" ],
+].forEach(test_support);
+
+// UPDATE BELOW to ensure the entry gets created
+
+// mark
+self.performance.mark('mymark');
+
+// measure
+self.performance.measure('mymeasure');
+
+// resource
+fetch(self.location.href + "?" + Math.random());
+
+// longtask

```

Comment:
I think for now longtasks are only supported in Window.

Reply:
indeed